solr/core/src/java/org/apache/solr/api/ContainerPluginsRegistry.java

```diff
@@ -372,9 +396,27 @@ public void init() throws Exception {
         holders.add(new ApiHolder((AnnotatedApi) api));
       }
     }
+
+    @SuppressWarnings("rawtypes")
```

Comment:
At the very least, don't suppress raw types here.

Reply:
it's not required